apaeval_env.yaml

```diff
@@ -12,4 +12,4 @@ dependencies:
   - mamba=0.13.0
   - tibanna=0.18.3
   - pip=21.1.2
-
+  - singularity=2.4.2
```

Comment:
I agree with @AsierGonzalez that it will be great if you can use the current `singularity` version. Do you mind checking whether replacing this with `- singularity=3.7.1` works on your side or not? If it works, I guess it doesn't hurt changing it to `- singularity=3.7.1`. If it doesn't work on your side, then I guess keeping `- singularity=2.4.2` is also fine.
Thank you!!